projects/components/src/popover/popover.ts

```diff
@@ -27,7 +28,7 @@ export interface PopoverHiddenPosition {
 
 export interface PopoverRelativePosition {
   type: PopoverPositionType.Relative;
-  origin: ElementRef;
+  origin: ElementRef | Element | Point;
```

Comment:
Separating changes like this and tab group out from the actual feature PR is usually helpful for reviewers. Here for example, we've added a new type to the providable positions, but do we actually handle the new type? I don't see that code

Reply:
So As we discussed earlier as well if we do not want to pass element itself we can pass Point instead on that, which is helpful.

Reply:
Yeah, Agree. I should have separated the PRs. Will surely look for future PRs.

Reply:
Where was this discussion? I don't recall it (but in fairness, I have horrible memory). How I see this being used is just `origin: nodes[index].getBoundingClientRect()` - but isn't `origin: nodes[index]` the same thing without an api change?

Reply:
Yeah, `nodes[index]` gives `Element` not `ElementRef`.
So inside the file `popover-position-builder.ts `at `line 30` we have `.flexibleConnectedTo(position.origin)`
In this function, origin is of `type: FlexibleConnectedPositionStrategyOrigin`
Which is ```
export declare type FlexibleConnectedPositionStrategyOrigin = ElementRef | Element | Point & {
    width?: number;
    height?: number;
};
```

Reply:
I just made it same. I did not want to pass the `Element`, so used `Point` instead

Reply:
> I did not want to pass the Element, so used Point instead

Why didn't you want to pass the element? Given that's what we're starting with and it doesn't look like we're modifying the point we give it, aren't the two equivalent?

Re: Element vs ElementRef

We only accepted an ElementRef here because angular code should not be working with native elements. The D3 code breaks that, but we want to encapsulate that to be contained only within the D3 code, rather than modifying other parts of the code to accept native elements. We can get from an element to element ref with a simpler wrapper `origin: new ElementRef(nodes[index])`

Reply:
Done!!

Reply:
Thanks! will let others review everything else 😉